### ISSUE TITLE
Temporal Loss - code vs. paper

### ISSUE THREAD

**Issue by OriLifschitz** [vccimaging/Reconstrution_Through_Moving_Water#6]
Hello, impressive work and masterfully written paper!

This is the code for the temporal loss:
` 
# temporal loss
temporal_loss = 0
Laplacian = torch.squeeze(C_x + C_y)
for i in range(1, time_frame - 1):
  temporal_loss += torch.sum(mask_cur[i:i + 1, 0, :, :] * (
          (height_map_cuda[i + 1, :, :] + height_map_cuda[i - 1, :, :] - 2 * height_map_cuda[i, :, :] - 0.5 * Laplacian[i, :, :]) ** 2)
  )
`
While this is the equation taken from the paper:
![image](https://github.com/user-attachments/assets/f069e797-df7b-4b37-9904-3ac1ad6f3c6c)

I am a bit confused regarding where did the constant 'c'  in eq. (13) ("c is the magnitude of the velocity") disappeared in the code please?
Or where in the paper can I read about this change between paper and code please?

Thanks a lot !
